bifrost/release_lock_test.go

```diff
@@ -1,10 +1,9 @@
 package bifrost
 
 import (
-	"testing"
-
 	"github.com/coinbase/step/utils/to"
 	"github.com/stretchr/testify/assert"
+	"testing"
```

Comment:
[Stdlib imports come first](https://github.com/golang/go/wiki/CodeReviewComments#imports).

Reply:
Agree, but with stylistic issues like these, we should probably opt for some automated linting or formatting.